packages/@core.keystone/plugins/softDeleted.js

```diff
@@ -78,11 +80,98 @@ const softDeleted = ({ deletedAtField = 'deletedAt', newIdField = 'newId' } = {}
     return { fields, hooks, access: newAccess, ...rest }
 }
 
-function applySoftDeletedFilters (access, deletedAtField) {
+/**
+ * Gets "where" variables from http graphQL request
+ * @param {Object} args
+ * @return {Object}
+ */
+function getWhereVariables (args) {
+    const method = get(args, ['context', 'req', 'method'])
```

Comment:
```suggestion
    // NOTE: it's a hack! Because we don't have access to GraphQL variables inside Keystone access function!
    const method = get(args, ['context', 'req', 'method'])
```